mlflow/transformers/llm_inference_utils.py

```diff
@@ -8,13 +22,133 @@
 _LLM_INFERENCE_TASK_COMPLETIONS = "llm/v1/completions"
 _LLM_INFERENCE_TASK_CHAT = "llm/v1/chat"
 
+_LLM_INFERENCE_OBJECT_NAME = {
+    _LLM_INFERENCE_TASK_COMPLETIONS: "text_completion",
+    _LLM_INFERENCE_TASK_CHAT: "chat.completion",
+}
+
 _SUPPORTED_LLM_INFERENCE_TASK_TYPES_BY_PIPELINE_TASK = {
     "text-generation": [_LLM_INFERENCE_TASK_COMPLETIONS, _LLM_INFERENCE_TASK_CHAT],
 }
 
+_SIGNATURE_FOR_LLM_INFERENCE_TASK = {
```

Comment:
Those signatures don't contain params schemas, then the params extracted from `preprocess_llm_inference_params` won't be able to work

Reply:
Regarding the signatures, because `preprocess_llm_inference_params` is done internally in `predict` inside the `transformers` flavor, the params extracted would be directly passed to `_predict` and the transformers pipeline directly which would work, and the added tests in `tests/transformers/test_transformers_model_export.py` have tested that both `max_tokens` and `stop` are effective.

If a user passes parameters to the `predict` function, they would be ignored though.